### ISSUE TITLE
Multi Level Properties

### ISSUE THREAD

**Issue by alfkonee** [OrleansContrib/Orleans.Indexing#1]
Hello there Please I'd like to ask if the Indexing works for MultiLevel Child Properties say
```
class CLS1{
 int Prop1{get;set}
 CLS2 Child{get;set}
}
class CLS2{
 [ActiveIndex]
 string data{get;set}
}
```
will this work if I use the CLS1 Class as a State for the IndexableGrain??

@sergeybykov

**Comment by TedHartMS** [comment]
No, we would only look for index attributes on properties of the specified Properties class.
